### PR TITLE
React bookstore styling

### PR DESCRIPTION
**Pull request by tinatsina**
## Summary
- used React default styling pattern
- update UI to match [Zeplin Template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
- Updated the `3`major components
    - Input Form
    - Book Component
    - Navbar Component

## Change Log
- `feature`  --> `development`

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -12,13 +12,29 @@ const Book = ({ bookTitle, bookAuthor, bookID }) => {
   };
 
   return (
-    <>
+    <li className="book-item-component">
```
Comment (brytebee) [inline src/components/Book.js:15]: - [ ] **Required:** The card components should be a look-alike as the essence of this task is to test your ability to produce a design template. 
- Kindly ensure every element title, author, book name, and buttons have the specified colors, font style and size.
- Include the bars - | - that exist between the edit, remove, and comment buttons.

Your Design | Template
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @tinatsina👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const Navbar = () => (
   <>
     <nav>
       <ul>
-        <li>BOOKSTORE</li>
+        <li className="logo-icon">BOOKSTORE CMS</li>
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply src/components/Navbar.js:11]: # Resolved 👍 
- I used your advice and made the necessary changes
- now looks more similar to each other
- the `avatar` has been added as well
- thanks for the assist 🙏

File: src/components/Book.js (hunk 1 above)
Comment (tinatsina) [reply src/components/Book.js:15]: # Resolved 👎 
- took your advice and resolved errors
- now the bars have also been included
- text format has been updated to match the template
- thanks for noticing

Horizon Node

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @tinatsina 

Your project is complete! Great job for making the required changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 


tufoin


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**